pinot-core/src/main/java/org/apache/pinot/core/operator/query/FilteredGroupByOperator.java

```diff
@@ -206,6 +210,20 @@ protected GroupByResultsBlock getNextBlock() {
         resultsBlock.setNumGroupsWarningLimitReached(numGroupsWarningLimitReached);
         return resultsBlock;
       }
+      if (GroupByUtils.shouldSortAggregateUnderSafeTrim(_queryContext)) {
```

Comment:
(minor) Make the code consistent with `GroupByOperator`. This way it is easier to extract common code in the future. We may also add a TODO to extract common code